docs/develop/yeoman-generator-overview.md

```diff
@@ -0,0 +1,102 @@
+
+When you have answered this question, the generator creates the project and installs the dependencies. You may see **WARN** messages in the npm output on screen. You can ignore these. You may also see messages that vulnerabilities were found. You can ignore these for now, but you will eventually need to fix them before your add-in is released to production. For more information about fixing vulnerabilities, open Bing and search for "npm vulnerability".
+
+If the creation has been successful, you will see a **Congratulations!** message in the command window, followed by some suggested next steps. (If you're using the generator as part of a quick start or tutorial, ignore the next steps in the command window and continue with the instructions in the article.)
+
+> [!TIP]
+> If you want to create the scaffolding of an Office Add-in project, but postpone the installation of the dependencies, add the `--skip-install` option to the `yo office` command. The following code is an example.
+>
+> ```command&nbsp;line
+> yo office --skip-install
+> ```
+>
+> When you're ready to install the dependencies, navigate to the root folder of the project in a command prompt and enter `npm install`.
+
+## Manifest-only option
+
+This option creates only a manifest for an add-in. The resulting project doesn't have a Hello World add-in, any of the scripts, or any of the dependencies. Use this option in the following scenarios:
+
+- You want to use different tools from the ones a Yeoman generator project installs and configures by default. For example, you want to use a different bundler, transpiler, task runner, or development server.
+- You want to use a web application development framework, other than Angular or React, such as Vue.
+
+See [Use Vue to build an Excel task pane add-in](../quickstarts/excel-quickstart-vue.md) for an example of using the generator with the manifest-only option.
+
+## Use command line parameters
+
+You can also add parameters to the `yo office` command. The two most common are the following:
+
+- `yo office --details`: This will output brief help about all of the other command line parameters.
+- `yo office --skip-install`: This will prevent the generator from installing the dependencies.
+
+For detailed reference about the command line parameters, see the readme for the generator at [Yeoman generator for Office Add-ins](https://github.com/officedev/generator-office).
+
+## Troubleshooting
+
+If you encounter problems using the tool, your first step should be to reinstall it to be sure that you have the latest version. If doing so doesn't fix the problem, search the [issues of the GitHub repo for the tool](https://github.com/OfficeDev/generator-office/issues) to see if anyone else has encountered the same problem and found a solution. If no one has, [create a new issue](https://github.com/OfficeDev/generator-office/issues/new?assignees=&labels=needs+triage&template=bug_report.md&title=).
```

Comment:
Consider linking back to https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/yeoman-generator-overview?branch=pr-en-us-3299#install-the-generator for guidance on reinstalling the tool.

Reply:
```suggestion
If you encounter problems using the tool, your first step should be to reinstall it to be sure that you have the latest version. (See [Install the generator](#install-the-generator) for details.) If doing so doesn't fix the problem, search the [issues of the GitHub repo for the tool](https://github.com/OfficeDev/generator-office/issues) to see if anyone else has encountered the same problem and found a solution. If no one has, [create a new issue](https://github.com/OfficeDev/generator-office/issues/new?assignees=&labels=needs+triage&template=bug_report.md&title=).
```